index.d.ts

```diff
@@ -0,0 +1,41 @@
+declare module 'react-swipeable' {
+  import * as React from 'react'
+
+  namespace ReactSwipeable {
+    interface Event<T> {
+      event: React.TouchEvent<T>
+      deltaX: number
+      deltaY: number
+      absX: number
+      absY: number
+      velocity: number
+      dir: 'Left' | 'Right' | 'Up' | 'Down'
+    }
+
+    type Callback<T extends Element = HTMLElement> = ({}: Event<T>) => void
+
+    interface SwipeableProps<T extends Element = HTMLElement> extends React.ClassAttributes<Swipeable<T>>, React.HTMLAttributes<T> {
+      // Event data
+      onSwiped?: Callback<T>
+      onSwipedLeft?: Callback<T>
+      onSwipedRight?: Callback<T>
+      onSwipedUp?: Callback<T>
+      onSwipedDown?: Callback<T>
+      onSwiping?: Callback<T>
+
+      // Configuration Props
+      delta?: number
+      preventDefaultTouchmoveEvent?: boolean
+      trackTouch?: boolean
+      trackMouse?: boolean
+      rotationAngle?: number
+
+      // Component Specific Props
+      nodeName?: string
+      innerRef?: React.Ref<T>
+      children?: React.ReactNode | React.ReactNode[]
+    }
```

Comment:
we could add `style` and `className` as they are passed to the inner node: https://github.com/dogfessional/react-swipeable/blob/ac7239652c5912d275c5b36193cdbb3ed2b3241d/src/index.js#L224-L227

Reply:
Yep, added